docs/verbs/catkin_build.rst

```diff
@@ -480,8 +480,20 @@ To run catkin tests for catkin packages, use the following:
 
 .. code-block:: bash
 
+    $ catkin run_tests --verbose -i
```

Comment:
`--verbose` should print all output from commands. `-i` prints the output out as it arrives, which when running multiple jobs causes interleaved output, hence `-i` being a short version of `--interleave`. `-p1 -i` would make sense I guess, since that would prevent interleaving of output by limiting it to one job.